docusaurus/docs-internals/editor-controls/data-controls/codeMirror.md

```diff
@@ -3,27 +3,44 @@ toc_max_heading_level: 4
+| `helper?.position` | `"top-start"` \ |   `"top"` \| `"top-end"` \| `"right-start"` \| `"right"` \| `"right-end"` \| `"bottom-end"` \| `"bottom"` \| `"bottom-start"` \| `"left-end"` \| `"left"` \| `"left-start"` \| `"top"` | Specifies the position of the tooltip relative to the helper icon.                                                                                                                                                                                                                                                                           |
+| `states?`          | `Array<State>`  | [`"normal"`]  | Allows for different styles based on the element's state <br/> <br/> <b>`State = "normal" \| "hover" \| "active"`</b> <br/> <br/> `"normal"` - the normal state of an element, <br/> `"hover"` - the state when the element is hovered over, <br/> `"active"` - the state when the element is active (e.g., current page in pagination).                                                                                         |
+| `config?.size`     | `"short"` \| `"medium"` \| `"large"` \| `"auto"` | `"medium"` | The size of textarea.                                                                                                                                                                                                                                                                                                                         |
+| `config?.language` | `"html"`  \|  `"css"`   \| `"markdown"`  \| `"xml"`  |   `"css"`   | Specify in which language the code will be written.                                                                                                                                                                                                                                                                                           |
+| `default?`         | `Default`       |       -       | The default control value. <br/> <br/> <b>`Default: { value: string; }`</b> <br/> <br/> `value` - the control's custom initial value <br/>                                                                                                                                                                                                                                                                                  |
+| `style?`           | `function`      |       -       | This function generates CSS output based on the value from the control. The parameter is an object containing a `value` key, which holds the current value of the control. The function returns an object with a CSS selector key and CSS property values.  <pre>`style: ({value}) => {`<br/> `return {`<br/>  `"{{WRAPPER}} .brz-text": {`<br/>   `color: value === "" ? "black" : "red"`<br/>  `}`<br/> `}`<br/>`}`</pre> |
```

Comment:
about example in `style` param

i think will be better if we will add an example with check if the control return some value then set display: block, if no, then display: none